<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LEGO</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveOBJ"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRenderingBasic"/>
    <addaction name="actionRenderingSSAO"/>
    <addaction name="actionRenderingHatching"/>
   </widget>
   <widget class="QMenu" name="menuSimplify">
    <property name="title">
     <string>Simplify</string>
    </property>
    <addaction name="actionInputVoxel"/>
    <addaction name="actionSimplifyByOpenCV"/>
    <addaction name="actionSimplifyByOurCustom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimplify"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenSliceImage">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Slice Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRenderingBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionRenderingSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
  <action name="actionRenderingHatching">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hatching</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveOBJ">
   <property name="icon">
    <iconset>
     <normaloff>Resources/save.png</normaloff>Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save OBJ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSimplifyByOpenCV">
   <property name="text">
    <string>OpenCV</string>
   </property>
  </action>
  <action name="actionSimplifyByOurCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionInputVoxel">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionSaveImageHD">
   <property name="text">
    <string>Save Image HD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
